rfcs/linux-arm64-support.md

```diff
@@ -0,0 +1,81 @@
+For backwards compatibility and to support the packages that already have been built for their respective architecture, the selection criteria will fall back to just platform name if the current architecture is the original default architecture.
+* **Platform Abstraction Layer (PAL) separation**<br>
+The PAL layer was designed to abstract platform specific cmake scripts and definitions based on only the platform name. There will be no changes to this architecture. Instead, anything that differentiates a platform by different architectures will be handled by the platform’s PAL script directly. The existing Linux-specific cmake scripts will make any adjustments that are architecture dependent within the script itself without the need of introducing yet another layer of abstraction. 
+* **CMake tool chain for Linux/ARM64**<br>
+There will be a new toolchain needed to build Linux on the ARM64 architecture. The toolchain may need to handle the case of native ARM64 compilation (e.g. building on an ARM64 host machine directly) and cross compiling (e.g. building using ARM64 cross compilers on an x64 based host platform).
+
+## What are the advantages of the feature?
+
+* Support for ARM64 based processors will provide cost saving benefits on cloud based workflows vs the traditional x86 architecture. Over time this could result in significant savings over applications such as multiplayer servers and hosting simulations.
+* Introducing the ability support multiple architectures for the same platform will set the framework for other multi-architecture platforms in the future.
+
+## What are the disadvantages of the feature?
+
+* This will add another platform support burden in the following areas
+
+  * **Build support (Build Nodes, AR, etc)** <br>
+  Every platform that is supported needs infrastructure setup to include it in the automated review process. This will increase the infrastructure usage cost and potentially increase the duration of pull requests. Also, adding new platforms to the AR process adds additionalk burden on the community and contributors to make sure their changes/contributions pass on that platform.
+
+  * **Feature Parity** <br>
+  Every feature that is supported on the current host platforms will need to be supported on the new architecture as well. This will increase automated and manual testing. (This can be mitigated by making Linux ARM64 as a target platform, ie disabling any editor or tools as part of the build)
+
+* All supported 3rd party libraries will need to be built on Linux ARM64 and packaged
+
+  * Every new 3rd Party library dependency must be built on an ARM64 architecture in addition to its x86 counterpart
+  * Risk of an external 3rd Party library not supporting ARM
```

Comment:
If we find a 3rdParty library that doesn't support ARM after building it, what would be the mitigation for that?

Reply:
If possible, gate the feature that depends on it with a trait. I've already found that squish-ccr is having this issue.I will add this point to the FAQ.